spec/factories/tasks.rb

```diff
@@ -0,0 +1,13 @@
+FactoryBot.define do
+  factory :task do
+    title { FFaker::Book.title }
+    description { FFaker::Lorem.paragraph }
+    date { FFaker::Time.between(1.year.ago, Date.today).to_date }
+    status { %w[todo progress done].sample }
```

Comment:
```suggestion
    status { Task.statuses.keys.sample }
```

Reply:
should I also try to apply this suggestion on the users.rb factory ? 
_code snippet from the user factory:_ 
`role { %w[user_role admin_role].sample }`

Reply:
Yes sure!